src/OIDCCryptUtils.purs

```diff
@@ -0,0 +1,48 @@
+module OIDCCryptUtils (RSASIGNTIME(), hashNonce, bindState, unbindState, verifyIdToken) where
```

Comment:
I think it's ok for initial pr. Maybe it should be released with `0.0.1` then. And `0.1.0` release could have all this `newtype` machinery. Could you please add an issue for this?